tests/test_forms.py

```diff
@@ -42,8 +41,6 @@ def test_valid_data(self):
                 "email": "testuserl@test.com",
                 "first_name": "Spencer",
                 "last_name": "Huang",
```

Comment:
It's likely that you will still need a password entry here...

Reply:
I think they work the same. This function is not called in any files now and in `forms.py` the field only requires email, first_name, and last_name. But I added the password entry in the most recent commit, and it passes as well.

Reply:
@enpuyou the newest commit doesn't pass, it fails with the same error... Perhaps you should coordinate with whoever put together the forms? Would that have been @barrezuetai or @aubreypc?

Reply:
Yes, I just saw that. It's weird that the test case passed in my local. I think this form is created by @barrezuetai. And I think I know why this is happening. Let me try something else see if it works...